packages/twenty-front/src/hooks/__tests__/usePageChangeEffectNavigateLocation.test.ts

```diff
@@ -134,7 +134,7 @@ const testCases = [
   { loc: AppPath.InviteTeam, isLoggedIn: true, subscriptionStatus: SubscriptionStatus.Active, onboardingStatus: OnboardingStatus.Completed, res: defaultHomePagePath },
 
   { loc: AppPath.PlanRequired, isLoggedIn: true, subscriptionStatus: undefined, onboardingStatus: OnboardingStatus.PlanRequired, res: undefined },
-  { loc: AppPath.PlanRequired, isLoggedIn: true, subscriptionStatus: SubscriptionStatus.Canceled, onboardingStatus: OnboardingStatus.Completed, res: undefined },
+  { loc: AppPath.PlanRequired, isLoggedIn: true, subscriptionStatus: SubscriptionStatus.Canceled, onboardingStatus: OnboardingStatus.Completed, res: defaultHomePagePath },
```

Comment:
to check with @charlesBochet and @martmull if it still makes sense

Reply:
I think we should redirect to '/settings/billing' in that case? Or 'undefined' seems like an acceptable option too. This is very theoretical, probably doesn't matter much, but one of the two options would be best imo

Reply:
no, we should not redirect in order to be able to pay for a new subscription, so undefined is the proper expected response